### PR TITLE
ci: revert the multistage builds

### PR DESCRIPTION
**Pull request by Nithos**
# Description

Reverting the Docker multi-stage build work. Seems that the CI is extremely slow then.

### CODE REVIEW

File: packages/services/hmi-server/docker/jvm/Dockerfile.jvm [hunk 1]
```diff
@@ -1,37 +1,93 @@
 ####
-# Compile and Build the Server using regular JVM
+# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
```
Comment (dgauldie) [inline packages/services/hmi-server/docker/jvm/Dockerfile.jvm:2]: we can remove all this boilerplate comments

File: packages/services/hmi-server/docker/jvm/Dockerfile.jvm [hunk 2]
```diff
@@ -1,37 +1,93 @@
...
-COPY --from=builder --chown=185 code/build/quarkus-app/*.jar /deployments/
-COPY --from=builder --chown=185 code/build/quarkus-app/app/ /deployments/app/
-COPY --from=builder --chown=185 code/build/quarkus-app/quarkus/ /deployments/quarkus/
+COPY --chown=185 build/quarkus-app/lib/ /deployments/lib/
+COPY --chown=185 build/quarkus-app/*.jar /deployments/
+COPY --chown=185 build/quarkus-app/app/ /deployments/app/
+COPY --chown=185 build/quarkus-app/quarkus/ /deployments/quarkus/
 
-EXPOSE 3000
+EXPOSE 8080
```
Comment (dgauldie) [inline packages/services/hmi-server/docker/jvm/Dockerfile.jvm:89]: should stay as 3000

File: packages/services/hmi-server/docker/native/Dockerfile.native [hunk 3]
```diff
@@ -1,31 +1,27 @@
...
-USER quarkus
-WORKDIR /code
-COPY packages/services/hmi-server/src /code/src
+# This Dockerfile is used in order to build a container that runs the Quarkus application in native (no JVM) mode.
+#
+# Before building the container image run:
+#
+# ./gradlew build -Dquarkus.package.type=native
+#
+# Then, build the image with:
```
Comment (dgauldie) [inline packages/services/hmi-server/docker/native/Dockerfile.native:8]: we can remove these comments

File: packages/services/hmi-server/docker/native/Dockerfile.native [hunk 4]
```diff
@@ -1,31 +1,27 @@
...
+###
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6-994
+WORKDIR /work/
+RUN chown 1001 /work \
+    && chmod "g+rwX" /work \
+    && chown 1001:root /work
+COPY --chown=1001:root build/*-runner /work/application
 
-RUN ./gradlew build -Dquarkus.package.type=native
+EXPOSE 8080
```
Comment (dgauldie) [inline packages/services/hmi-server/docker/native/Dockerfile.native:24]: leave as 3000

**Review by dgauldie — CHANGES_REQUESTED** [review]

File: packages/services/hmi-server/docker/jvm/Dockerfile.jvm (hunk 2 above)
Comment (Nithos) [reply packages/services/hmi-server/docker/jvm/Dockerfile.jvm]: done

File: packages/services/hmi-server/docker/native/Dockerfile.native (hunk 4 above)
Comment (Nithos) [reply packages/services/hmi-server/docker/native/Dockerfile.native]: done

File: packages/services/hmi-server/docker/native/Dockerfile.native (hunk 3 above)
Comment (Nithos) [reply packages/services/hmi-server/docker/native/Dockerfile.native]: done

File: packages/services/hmi-server/docker/jvm/Dockerfile.jvm (hunk 1 above)
Comment (Nithos) [reply packages/services/hmi-server/docker/jvm/Dockerfile.jvm:2]: done

**Review by dgauldie — APPROVED** [review]
LGTM

**Pull request merged**